src/utils/mapLinks.js

```diff
@@ -1,99 +1,85 @@
 export const displayLink = (room, campus) => {
-  if (!Object.is(room, undefined)) {
+  if (room !== undefined) {
     if (
-      Object.is(campus, 'Internet') ||
-      Object.is(campus, 'Off-Campus - Domestic') ||
-      Object.is(campus, 'Macomb') ||
-      Object.is(campus, 'Mt. Clemens') ||
-      Object.is(room.trim(), 'N/A')
+      campus === 'Internet' ||
+      campus === 'Off-Campus - Domestic' ||
+      campus === 'Macomb' ||
+      campus === 'Mt. Clemens' ||
+      room.trim() === 'N/A'
```

Comment:
I think you could simplify this inner if statement with just returning the value of all the checks like so

```js
export const displayLink = (room, campus) => {
  if (room !== undefined) {
    return (campus !== 'Internet' ||
      campus !== 'Off-Campus - Domestic' ||
      campus !== 'Macomb' ||
      campus !== 'Mt. Clemens' ||
      room.trim() !== 'N/A')
  }
}
```

Also, just out of curiosity if `room === undefined` you never return anything? is that how it's supposed to be?

Reply:
I guess so. That's how its been for all these years, and we've never seemed to have an issue with it